libvcx/src/aries/handlers/proof_presentation/verifier/state_machine.rs

```diff
@@ -87,22 +87,8 @@ impl VerifierSM {
             VerifierState::Initiated(state) => {
                 match message {
                     VerifierMessages::SendPresentationRequest => {
-                        let my_did = get_pw_did(connection_handle)?;
-                        let remote_did = get_their_pw_verkey(connection_handle)?;
-
-                        let presentation_request: PresentationRequestData =
-                            state.presentation_request_data.clone()
-                                .set_format_version_for_did(&my_did, &remote_did)?;
```

Comment:
So I've checked this and I see that this method, based on format of DIDs sets up "format" of the proof request - either V1 or V2. If we remove this, what `format` will be used? Do you know what's difference between V1/V2?

Reply:
It's format version for DID, not for the proof request and as far as I am aware, it is internal to aries-vcx and never read (only set by verifier), so `set_format_version_for_did` can be safely deleted as well. But perhaps it refers to some DID-spec / aries standard.